.changeset/kind-pens-deliver.md

```diff
@@ -0,0 +1,13 @@
+---
+'@backstage/backend-common': minor
+'@backstage/cli': minor
+'@backstage/techdocs-common': minor
+'@backstage/plugin-auth-backend': minor
+'@backstage/plugin-catalog': minor
+'@backstage/plugin-kafka-backend': minor
+'@backstage/plugin-kubernetes-backend': minor
+'@backstage/plugin-scaffolder-backend': minor
+'@backstage/plugin-techdocs-backend': minor
```

Comment:
Normally I'd say that these should be `patch` since no functionality is changed, but actually since the content type of the response changes in some places ...

Could you make sure to mention that in the text below? Something like "Note that this means that content types now more consistently are `application/json` on backend responses, even in error cases."

Reply:
Tbh I think we should go with `patch`, the cases were we switch from `.send({...})` -> `.json({...})` don't change anything since send already detects the object and sets the content type. The other ones I'd argue are just compatible improvements xP

The `@backstage/cli` also doesn't have any changes, so should be removed from the list